js/src/wp-seo-post-scraper.js

```diff
@@ -96,7 +97,7 @@ import { setMarkerStatus } from "./redux/actions/markerButtons";
 	 * @returns {boolean} True when markers should be shown.
 	 */
 	function displayMarkers() {
-		return wpseoPostScraperL10n.show_markers === "1";
+		return ! isGutenbergPostAvailable() && wpseoPostScraperL10n.show_markers === "1";
```

Comment:
rendundant space after !

Reply:
Hi @aleksanderneroda, Thank you very much for your code review. It's always great to see people from the community contributing to our code.  However, the space before the exclamation mark is in line with our code style rules, which can be found here: https://github.com/Yoast/eslint/blob/master/default.yml#L203.